resources/styles/components/exercise-card.less

```diff
@@ -18,51 +14,55 @@
     background-color: @selected-exercise-background;
   }
 
-  .panel-heading {
-    position: absolute;
-    top: 0;
-    right: 0;
-    bottom: 0;
-    left: 0;
-    display: none;
-    background-color: fade(@tutor-neutral-lighter, 0%);
-    border: none;
-
-    .add-or-remove {
-      position: absolute;
-      top: 50%;
-      left: 50%;
-      height: 100px;
-      width: 100px;
-      margin-top: -50px;
-      margin-left: -50px;
-      background-color: fade(@tutor-white, 85%);
-      border: 1px solid @tutor-neutral-lighter;
-      border-radius: 100px;
-      color: @tutor-neutral;
-
-      &::before {
-        .fa-icon();
-        content: @fa-var-plus;
-        font-weight: 300;
-        text-align: center;
-        font-size: 50px;
-        line-height: 100px;
-        width: 100px;
-        position: relative;
+  &.is-selectable {
+    .panel-body {
+      position: relative;
+      &:hover {
+        .toggle-mask {
+          opacity: 0.85;
+          visibility: visible;
+        }
       }
+    }
 
-      &.active {
+    .toggle-mask {
+      cursor: pointer;
+      position: absolute;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      left: 0;
+      visibility: hidden;
+      z-index: 1;
+      opacity: 0;
+      transition: opacity .25s ease-in-out;
+      display: flex;
+      border: none;
+      justify-content: center;
+      align-items: center;
+
+      .message {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        font-size: 2.5rem;
+        font-weight: 700;
         &::before {
-          content: @fa-var-minus;
+          .fa-icon();
+          content: @fa-var-plus;
+          font-weight: 300;
+          text-align: center;
+          font-size: 50px;
+          border-radius: 60px;
+          width: 60px;
+          height: 60px;
+          padding-top: 5px;
         }
       }
     }
-  }
-
-  &:hover {
-    .panel-heading {
-      display: block;
+    &.is-selected .toggle-mask .icon::before {
```

Comment:
Another great catch.   `:hover` wasn't needed but it should have been `.message` instead of `.icon` (`.icon` was the old element, before the text message was added).

However, they're actually both incorrect :(  When I re-looked at the mockup the styles are switched from what they are on HW builder.  It should show the minus when they're selected since a selection means they're being removed, and then a plus on de-select.
 
I think i'll expand the `.exercise-card-hover-colors` mixin to be a general `.exercise-card-hover-styles` that will also accept icon arguments (with defaults.